apps/dynatrace/base/kustomize.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: dynatrace
+  namespace: flux-system
+spec:
+  path: ./apps/dynatrace/${ENVIRONMENT}/base
```

needs to depend on the crd https://github.com/hmcts/sds-flux-config/blob/master/apps/dynatrace/base/kustomize.yaml#L8-L9

and you should have an interval set
```suggestion
  interval: 10m
  path: ./apps/dynatrace/${ENVIRONMENT}/base
  dependsOn:
    - name: dynatrace-crd
```

---

Fixed, thanks for pointing this out.